packages/forklift-console-plugin/src/plans/create/CreatePlanWizard.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { useForkliftTranslation } from 'src/utils';
+
+import { Form, Title, Wizard, WizardStep, WizardStepType } from '@patternfly/react-core';
+
+import { GeneralInformationForm } from './steps/GeneralInformationForm';
+import { defaultCurrentStep, planStepIndexes, planStepNames, PlanWizardStepId } from './constants';
+import { CreatePlanWizardFooter } from './CreatePlanWizardFooter';
+
+import './CreatePlanWizard.style.scss';
+
+export const CreatePlanWizard: FC = () => {
+  const { t } = useForkliftTranslation();
+  const form = useForm({ mode: 'onChange' });
+  const [currentStep, setCurrentStep] = useState<WizardStepType>(defaultCurrentStep);
+  const { formState, getValues } = form;
+  const formValues = getValues();
```

Comment:
BTW if IRC , getValues arent updates on values changes - they have watch and useWatch hook for that

Reply:
This is only used to display temp values in the review page right now, but sure I can switch it to `watch`.